qtpy/QtSql.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright © 2009- The Spyder Development Team
+#
+# Licensed under the terms of the MIT License
+# (see LICENSE.txt for details)
+# -----------------------------------------------------------------------------
+"""Provides QtSql classes and functions."""
+
+# Local imports
+from . import PYQT4
```

Comment:
This could be just one import:

`from . import PYQT5, PYSIDE2, PYQT4, PYSIDE, PythonQtError`

Reply:
I personally prefer the way as it is in my commit. But it's not my project and if it's common here there's no point of argue => it will be in my next update

Reply:
I'm not sure, I think there's code styles guidelines for qtpy, @goanpeca ?